.github/workflows/node-flow-pull-request-checks.yaml

```diff
@@ -295,6 +318,33 @@ jobs:
       slack-api-token: ${{ secrets.PLATFORM_SLACK_API_TOKEN }}
       grafana-agent-username: ${{ secrets.GRAFANA_AGENT_USERNAME }}
       grafana-agent-password: ${{ secrets.GRAFANA_AGENT_PASSWORD }}
+
+  service-test-panel:
+    name: Service Test Panel
```

Comment:
Maybe we should call this the `Hedera Node JRS Panel`?

Reply:
Changed